src/Window.cc

```diff
@@ -2472,7 +2472,7 @@ void FluxboxWindow::buttonPressEvent(XButtonEvent &be) {
 
 }
 
-const unsigned int DEADZONE = 4;
+const int DEADZONE = 4;
```

Comment:
Since the deadzone cannot be < 0, the proper fix is to cast the compared values.